lib/JiraClient.ts

```diff
@@ -0,0 +1,51 @@
+import fetch = require("node-fetch");
+
+const JIRA_DOMAIN = 'https://sonarsource-sandbox-608.atlassian.net';
+
+export class JiraClient {
+
+    constructor(private readonly jiraToken: string) { }
```

Comment:
This will need two parameters, and the base64 magic should happen in this contructor